### PR TITLE
add other learning resources. minor text edits.

### PR DESCRIPTION
**Pull request by seesolve**
Add other online learning sites and recent books.

### CODE REVIEW

File: docs/LEARNING.md [hunk 1]
```diff
@@ -5,5 +5,14 @@ commonly taught [object oriented approach](https://en.wikipedia.org/wiki/Object-
 learning Haskell can feel different than simply picking up another language.
 
 Fortunately there are numerous resources which presume some programming knowledge to begin with, such
-as the popular [Learn You a Haskell For Great Good!](http://learnyouahaskell.com/) In addition, you may refer to the Haskell
-[wiki](https://wiki.haskell.org/Learning_Haskell#Online_tutorials) for an abundance of resources on beginning.
+as 
+  - the popular [Learn You a Haskell For Great Good!](http://learnyouahaskell.com/) 
+  - University of Glasgow's [Functional Programming in Haskell](https://www.futurelearn.com/courses/functional-programming-haskell) course
+  - FP Complete's [School of Haskell](https://www.schoolofhaskell.com/)
+  - and the [Happy Learn Haskell Tutorial](www.happylearnhaskelltutorial.com).
```
Comment (rbasso) [inline docs/LEARNING.md]: The missing `http://` seems to make the link be interpreted as a relative path.

**Comment by seesolve** [comment]
Made the changes. Do I submit another pull request?

**Comment by rbasso** [comment]
> Made the changes. Do I submit another pull request?

Please, don't! :smile: 

Now you can squash both commits in a single one - they belong together - if you want to practice some git, or I can squash it while merging.

What do you prefer?

**Comment by rbasso** [comment]
In case you need some directions on how to *squash* these two commits in a single one, I just found this link that explain it really well:
http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

**Comment by seesolve** [comment]
I definitely want to squash 'em! Thanks for the link.

**Comment by seesolve** [comment]
I am so confused. That link talks about how to rebase from the command line. I thought it would be quicker and simpler to just use the GitHub interface following the instructions at [Merging a pull request using the GitHub web interface](https://help.github.com/articles/merging-a-pull-request/), but I don't see a Merge pull request button anywhere. Am I missing something?

If I need to do this via the command-line, I can complete this tomorrow. (And I'd like to complete this. The more practice I get with pull requests, the better.)

**Comment by rbasso** [comment]
> I am so confused.

#### Don't panic :smile: 

Five months ago I had no idea about how to do it, but don't worry. It's easier than it seems.

> I thought it would be quicker and simpler to just use the GitHub interface following the instructions at Merging a pull request using the GitHub web interface, but I don't see a Merge pull request button anywhere

You opened the *pull request*, but you would need *write* privileges in this repository to merge it. This is why I'll merge it for you, and I have an option to squash it while merging.

That said, it is usually a recommended practice to *rebase* in *master* and *squash* your commits before opening a pull request, unless there is some reason to have multiple commits, each one as a self-contained change.

Considering that this is a place for learning and people need to practice *git* somewhere, I usually suggest contributors to take the opportunity and develop some *git* muscles. 

> If I need to do this via the command-line, I can complete this tomorrow. (And I'd like to complete this. The more practice I get with pull requests, the better.)

No hush! Try to have fun in the process. :smile:

**Comment by seesolve** [comment]
Is there any way for me to rebase and squash on GitHub? I understand I don't have write permission to this repo, but when I forked to make changes, it created a repo under my profile. I don't, however, see a way to squash and merge there.

If not, then do I need to clone my forked repo to my computer to rebase and squash on the command line?

**Comment by rbasso** [comment]
> Is there any way for me to rebase and squash on GitHub?

In this case, I don't think it is possible. But I can squash/merge here if you prefer.

> If not, then do I need to clone my forked repo to my computer to rebase and squash on the command line?

Yes!

I would be something like...

#### Clonning 

```
git clone https://github.com/seesolve/xhaskell
cd xhaskell
```

#### Adding an upstream remote and syncing your master branch (optional)

```
git remote add upstream https://github.com/exercism/xhaskell
git pull upstream master
```

#### Rebasing

To start the interactive rebase, type:

```
git checkout patch-1
git rebase -i master
```

Remember to change `pick` in the second line to `squash`!

#### Pushing the changes back to your fork

Here you have to you `--force` or `-f` because the push will overwrite the `patch-1` branch in your fork.

```
git push -f
```

This will automatically be reflect in the PR.

#### The End

Seems a lot, right?

You will not have to this every time, and as soon as you get used to these commands, things will start to make more sense and it will become almost automatic to contribute.

**Comment by seesolve** [comment]
Know anything about using SSH keys?

**Comment by rbasso** [comment]
No. Why?

**Comment by seesolve** [comment]
Trying to clone the repo using SSH.

**Comment by seesolve** [comment]
Nevermind. Found out what I needed to get SSH to work right.

**Comment by seesolve** [comment]
Thanks for the instructions. Eli Bendersky's instructions are great with the overall explanation, but not so great with the step-by-step instructions. Definitely not for newbies. Your instructions were much better. I'll keep them for reference until they become habit.

**Comment by rbasso** [comment]
Now, feel free to remove you `patch-1` branch if you want to. It will not be used anymore. :smile:

**Comment by seesolve** [comment]
Thank you! I was wondering about that.

**Pull request merged**